Workbooks/WorkbooksMetadata.json

```diff
@@ -479,5 +479,18 @@
     "templateRelativePath": "OneIdentity.json",
     "subtitle": "",
     "provider": "One Identity LLC.​"
-  }
+  },
+  {
+    "workbookKey": "AzureSentinelSecurityAlerts",

```

Comment:
Our convention is to add the word "Workbook" in the end of the workbook key.
Look at the other workbook keys.